modules/samples/product/JmsTTLSample/src/org/sample/jms/SampleQueueSender.java

```diff
@@ -0,0 +1,65 @@
+package org.sample.jms;
+
+import javax.jms.DeliveryMode;
+import javax.jms.JMSException;
+import javax.jms.Queue;
+import javax.jms.QueueConnection;
+import javax.jms.QueueConnectionFactory;
+import javax.jms.QueueSession;
+import javax.jms.TextMessage;
+import javax.naming.Context;
+import javax.naming.InitialContext;
+import javax.naming.NamingException;
+import java.util.Properties;
+
+public class SampleQueueSender {
+
+    public static final String QPID_ICF = "org.wso2.andes.jndi.PropertiesFileInitialContextFactory";
+    private static final String CF_NAME_PREFIX = "connectionfactory.";
+    private static final String QUEUE_NAME_PREFIX = "queue.";
+    private static final String CF_NAME = "qpidConnectionfactory";
+    String userName = "admin";
+    String password = "admin";
+    private static String CARBON_CLIENT_ID = "carbon";
+    private static String CARBON_VIRTUAL_HOST_NAME = "carbon";
+    private static String CARBON_DEFAULT_HOSTNAME = "localhost";
+    private static String CARBON_DEFAULT_PORT = "5672";
+    String queueName = "testQueue";
```

Comment:
Best to change the queue name in both subscriber and publisher. Since the same name is being used in another sample as far as I remember.